src/scripts.js

```diff
@@ -0,0 +1,72 @@
+ loadJSON(function(response) {
+    var items = JSON.parse(response);
+	new Vue({
+		data: {
+			filter: ''
+		},
+		computed: {
+			data() {
+				return items.entries;
+			}
+		},
+		methods: {
+			filtered(item) {
+				let show = true;
+				
+				if(this.filter.length) {
+					
+					show = false;
+					
+					let filterKeyword = this.filter.toLowerCase();
+					
+					Object.keys(item).map(function(key) {
+						if(typeof item[key] === 'string') {
+							let value = item[key].toString().toLowerCase();
+							
+							if(value.includes(filterKeyword)) {
+								show = true;
+								$( "th" ).addClass( "lol" );
+							}
+						}
+						
+					});
+				}
+				return show;
+			}
+		}
+	}).$mount('#app');
+});
+
+function loadJSON(callback) {   
+    var xobj = new XMLHttpRequest();
+        xobj.overrideMimeType("application/json");
+    xobj.open('GET', 'https://raw.githubusercontent.com/toddmotto/public-apis/master/json/entries.min.json', true);
+    xobj.onreadystatechange = function () {
+          if (xobj.readyState == 4 && xobj.status == "200") {
+            // Required use of an anonymous callback as .open will NOT return a value
+            // but simply returns undefined in asynchronous mode
+            callback(xobj.responseText);
+          }
+    };
+    xobj.send(null);  
+ }
+
+function filterRows() {
```

Comment:
I have a feeling this can be done with Vue itself. Leave it with me, I'll pull down the branch and have a play later.